### ISSUE TITLE
Can I use inlet Euclid on Turtlebot3? Has anyone done this before? 

### ISSUE THREAD

**Issue by stevensu1838** [ROBOTIS-GIT/turtlebot3#381]
Planning buying a intel Euclid kit to replace the raspberry Pi? Have any of you done this before? Can I use the pre-installed ROS kinetic on Euclid kit with Turtlebot3? Does it (USB3.0)works well with OpenCR(USB2.0)? I need your confirmation to make the decision to buy one. Cheers

**Comment by routiful** [comment]
Sorry, we don't  have any plan to do it.
But I assume that if Intel has support Euclid in ROS, you can try that what your purposes.

**Comment by BruceLun** [comment]
@stevensu1838 
I have a Euclid. I found the installed ROS packages are basically "static" and cannot be updated. If you configure the sources.list per ROS instructions then pull pull packages from the repository per usual, you will kill the camera driver (a custom driver is being used). It gets overwritten by the general driver from the ROS distro. I've tried MANY different ways to make this work with no luck. Each time had to re-install the OS image to the Euclid to recover. If you want to manually install TB3 packages from source, that may work but there could be dependency issues due to the existing packages being a particular version. Since there was no easy way to upgrade ROS packages, I basically gave up. Battery didn't last very long either.
